server/routes.go

```diff
@@ -955,6 +962,54 @@ func (s *Server) CreateBlobHandler(c *gin.Context) {
 	c.Status(http.StatusCreated)
 }
 
+func (s *Server) isLocal(c *gin.Context) bool {
+	if authz := c.GetHeader("Authorization"); authz != "" {
```

Comment:
Is auth required here (vs checking for )? I think there are simpler ways:

1. Check `127.0.0.1` or `::1`, verify `Origin` is not set (for CORS)
2. Drop a random secret string in a known location `/tmp/<random id>` directory and send it in the request. Server verifies it before reading the file